testnets/symphonytestnet/assetlist.json

```diff
@@ -24,13 +22,97 @@
       "display": "mld",
       "symbol": "MLD",
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/coin_icon.png"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/mld.png"
       },
       "images": [
         {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/coin_icon.png"
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/mld.png"
+        }
+      ]
+    },
+    {
+      "description": "Harmonized USD",
+      "denom_units": [
+        {
+          "denom": "uusd",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "husd",
+          "exponent": 6,
+          "aliases": ["harmony USD"]
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "uusd",
+      "name": "Harmony USD",
+      "display": "husd",
+      "symbol": "HUSD",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/husd.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/husd.png"
+        }
+      ]
+    },
+    {
+      "description": "Harmonized HKD",
+      "denom_units": [
+        {
+          "denom": "ukhd",
```

Comment:
everywhere ielse it seems to be HKD, but in the base and display denoms, it KHD. Just want to make sure there isn't a typo.

Reply:
it is supposed to be hkd, yes.  unfortunately, the dev did have a typo and that is out on the current version of the testnet.  it'll be amended before mainnet and when this moves out of config into something a bit easier to manage dynamically